README.md

```diff
@@ -1369,7 +1369,7 @@ As an optional second parameter you can pass the following options
 
 - `before`: a preprocessing function, gets called before nock.define
 - `after`: a postprocessing function, gets called after nock.define
-- `afterRecord`: a postprocessing function, gets called after recording. Is passed the array of scopes recorded and should return the array scopes to save to the fixture
+- `afterRecord`: a postprocessing function, gets called after recording. Is passed the array of scopes recorded and should return the array scopes as an object or stringified to save to the fixture
```

Comment:
Could this documentation be clarified? It’s confusing to refer to an array as an object. I think this means, return the intact array, a modified version of the array, or if custom formatting is desired, a stringified version of the array.

Reply:
@paulmelnikow Thank you for the input. I updated the `REAMDE`. Let me know if there are anymore changes you would like me to make. Cheers.